main.js

```diff
@@ -100,7 +125,7 @@ function timePenalty() {
     makeScreenFlex('#penaltyScreen');
 
     setTimeout(() => {
-        makeScreenFlex('#play_area');
+        makeScreenGrid('#play_area');
```

Comment:
I don't think is what you want anymore either. The screen bugs out after a time penalty.

Reply:
Good spot!